crates/chain/src/tx_graph.rs

```diff
@@ -1250,6 +1285,8 @@ pub struct ChangeSet<A = ()> {
     pub txouts: BTreeMap<OutPoint, TxOut>,
     /// Added anchors.
     pub anchors: BTreeSet<(A, Txid)>,
+    /// Added first-seen unix timestamps of transactions.
+    pub first_seen: BTreeMap<Txid, u64>,
```

Comment:
For backward compatibility, please add this field at the end of the struct and annotate it with:

```rust
#[cfg_attr(feature = "serde", serde(default))]
```

Reply:
The order we introduce fields matter because the order matters for some serialization formats.

`serde(default)` is important so that `serde` does not error out when this field is missing, instead puts the `Default` impl in it's place.